tests/scenarios/PowershellScriptTest/config.json

```diff
@@ -51,7 +57,8 @@
 			"startScript" :
 			{
 				"scriptPath" : "SayHello.ps1",
-				"runInVirtualEnvironment": true
+				"runInVirtualEnvironment": true,
+				"wait": true

```

Comment:
>true [](start = 12, length = 4)

New batch of tests for wait:false?

Reply:
Can't test with wait = false because the test framework does not account for scripts running async.

---
In reply to: [316826826](https://github.com/microsoft/MSIX-PackageSupportFramework/pull/79#discussion_r316826826) [](ancestors = 316826826)

Reply:
Can you quickly describe how wait:false and timeout were verified?

---
In reply to: [316940678](https://github.com/microsoft/MSIX-PackageSupportFramework/pull/79#discussion_r316940678) [](ancestors = 316940678,316826826)

Reply:
I ran my own application with wait and timeout.

---
In reply to: [317288022](https://github.com/microsoft/MSIX-PackageSupportFramework/pull/79#discussion_r317288022) [](ancestors = 317288022,316940678,316826826)